File: test/client_code/tests/packages/echo/echo.py
"""
packages/echo/echo.py
"""


def main(use, *args, **kwargs):
    """Tests echo package."""

    console, document, log, meta = use.console, use.document, use.log, use.meta

    echo = use("@@/echo/", test=True)
    result = echo(42)
    log("result:", result)

File: test/client_code/tests/packages/echo/_history/20260417t1615/echo.js
/*
packages/echo/echo.js
*/

export default async () => {
  console.clear();
  console.log("Tests echo package.")

  //
  //
  const log = await use("@@/console:log");
  log("Tests echo package.", {native: false})
  //
  //

  await (async () => {
    const echo = await use("@@/echo/", { test: true });
    const result = echo(42);
    console.log("result:", result);
  })();

  // Alt
  await (async () => {
    const echo = await use.packages("@@/echo/", true);

    //console.log("echo:", echo);
    //console.dir(echo);
    //const unwrapped = echo.unwrap();
    //console.log("unwrapped:", unwrapped);

    const result = echo({ foo: 42 }, 1, 2, 3);
    console.log("result:", result);
  })();
};


